### DISCUSSION TITLE
Is there a way to tell which files were actually removed by --strip-blobs-bigger-than?

### DISCUSSION THREAD

**Discussion by samal-rasmussen** [newren/git-filter-repo#558] [Q&A] [answered]
`git filter-repo --strip-blobs-bigger-than 10M` is great and it seems to have removed the large file I wanted it to remove, but I am unsure if it removed anything else and the commands doesn't tell me exactly what it removed.

Is there a way to tell which files it actually removed?

**Comment by me-and** [comment, marked as answer]
If you run `git rev-list --all --objects --filter=object:type=blob`, this will list all files in a repository, throughout its history, with both the file hash and the path. That means you can run that command before and after running your `git filter-repo` command, and compare the output; you should find that the only changes are the files that have been deleted are not listed in the rewritten repository.
